specs/_features/whisk/beacon-chain.md

```diff
@@ -382,49 +367,30 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
 
 ### Deposits
 
+```python
+def get_initial_whisk_k(validator_index: ValidatorIndex, counter: int) -> BLSFieldElement:
+    return BLSFieldElement(bytes_to_bls_field(hash(uint_to_bytes(validator_index) + uint_to_bytes(uint64(counter)))))
+```
+
 ```python
 def get_unique_whisk_k(state: BeaconState, validator_index: ValidatorIndex) -> BLSFieldElement:
     counter = 0
     while True:
         # hash `validator_index || counter`
```

Comment:
That comment should likely be moved to the new func

Reply:
fixed